rules_of_procedure-general_meeting.md

```diff
@@ -0,0 +1,84 @@
+# Rules of Procedure of the General Meeting of the Digital Engineering Student Body
+
+*This translation is for information purposes only and is not legally binding.*
+
+These rules of procedure were adopted by the general assembly on xx.xx.xxxx.
+
+## § 1 Scope
+
+The Rules of Procedure apply to the General Meeting of the Digital Engineering Student Body at the University of Potsdam.
+
+## § 2 Agenda
+
+(1) The General Meeting follows an agenda.
+
+(2) The agenda must be part of the timely invitation to a General Meeting.
+
+(3) Up to 14 days prior a General Meeting, motions can be submitted at the inviting persons for inclusion in the agenda.
+
+(4) Up to 7 days before a General Meeting, the final agenda must be announced in the same way as the invitation to the General Meeting.
+
+## § 3 Chairing of the Meeting
+
+(1) The General Meeting is chaired by the persons inviting.
+
+(2) The chairing of the meeting can be handed over to other members of the student body for individual items on the agenda.
+
+(3) If necessary, the chair may keep a list of speakers and/or limit the speaking time.
+
+(4) The chair decides how votes are to be counted or estimated and whether they are to be recorded afterwards.
+
+## § 4 Minutes
+
+(1) The General Meeting shall document its decisions in minutes.
+
+(2) The chair of the meeting shall appoint a member of the student body to take the minutes.
+
+(3) The minutes shall be made available to the student council no later than 7 days after the General Meeting.
+
+## 5 Motions
+
+(1) Motions must be written and submitted to the inviting persons within 14 days of receipt of the invitation to the General Meeting.
+
+(2) The inviting persons decide on the inclusion of the motion in the agenda.
+
+(3) Motions may be withdrawn by the person submitting the motion before they are dealt with.
+
+## § 6 Amendments
+
+(1) Amendments shall be dealt with before the respective motion is put to the vote.
+
+(2) Amendments may be accepted directly by the person making the original motion. If this does not happen, the General Meeting shall vote on the acceptance.
+
+(3) Amendments may be announced and justified in writing before the General Meeting.
+
+## § 7 Points of Order
+
+(1) Points of order shall be raised by raising both arms. This may be deviated from if the intention is clearly recognisable to the chairing of the meeting.
+
+(2) Points of order shall be dealt with immediately.
+
+(3) Before the vote, one person entitled to submit a motion may speak for and one against the point of order.
+
+(4) Points of order shall require a two-thirds majority of the members of the General Meeting present.
+
+(5) Motions of order are:
+  1. Motion to postpone an item on the agenda (simple majority),
+  2. Motion to close the list of speakers,
+  3. Motion to close the discussion and, if necessary, to take an immediate vote,

```

Comment:
Trivia: such motion is called "motion to order the previous question" in formal parlamentarian English ;)